Userland/Libraries/LibManual/PageNode.cpp

```diff
@@ -26,4 +27,10 @@ ErrorOr<String> PageNode::path() const
     return TRY(String::formatted("{}/{}.md", TRY(m_section->path()), m_page));
 }
 
+ErrorOr<NonnullRefPtr<PageNode>> PageNode::help_index_page()
+{
+    static NonnullRefPtr<PageNode> const help_index_page = TRY(try_make_ref_counted<PageNode>(sections[7 - 1], TRY(String::from_utf8("Help-index"sv))));
```

Comment:
Eh... Where does this error go? Is it returned to the caller on first init since it's a function static? 🤔. If this works that's cool but I'm surprised by the syntax.

Reply:
It would only fail for OOM situations, where we would probably quit the program anyways because some other thing crashes us. I still think it's nicer to propagate errors out of libraries (afaik there's no error swallowing in LibManual at all) even if we propagate them through serenity_main in the application.

Reply:
Clarification: all these TRYs are allocation-related